main.tf

```diff
@@ -0,0 +1,76 @@
+locals {
+  alarm_topic = "arn:aws:sns:${var.aws_region}:${var.aws_account_id}:${module.sns_label.id}-alarm"
```

Comment:
we should make this var.alarm_topic like here: https://github.com/justtrackio/terraform-aws-sns-topic/blob/main/main.tf#L121